python/setup.py

```diff
@@ -300,9 +300,9 @@ def append_cmake_bool(value, varname):
 
     def _run_cmake(self, pyarrow_cpp_home):
```

Comment:
By the way, do we have a JIRA to consolidate those two CMake-running method?

Reply:
Yes we do, it will be a part of this work: https://issues.apache.org/jira/plugins/servlet/mobile#issue/ARROW-17838